mustang_core/lib/src/store/mustang_store.dart

```diff
@@ -38,6 +38,13 @@ class MustangStore {
     return null;
   }
 
+  static MustangState? getState<MustangState>({String? name}) {
```

Comment:
This method always accepts string as arg and returns MustangState? as return value. It is redundant to generify